src/packages/core/property-editor/uis/tree-picker/property-editor-ui-tree-picker.element.ts

```diff
@@ -36,12 +39,19 @@ export class UmbPropertyEditorUITreePickerElement extends UmbLitElement implemen
 	@state()
 	ignoreUserStartNodes?: boolean;
 
+	#dynamicRoot?: UmbTreePickerSource['dynamicRoot'] | undefined;
+
+	#dynamicRootRepository: UmbDynamicRootRepository;
```

Comment:
You don't need to specify the type here if it's assigned in the constructor. I believe you could even new it up here directly.

```suggestion
	#dynamicRootRepository;
```